src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,42 +1,43 @@
 package edu.kis.vh.nursery;
 
-public class DefaultCountingOutRhymer {
-
-	private static final int STACK_CAPACITY = 12;
-
-	private static final int EMPTY_STACK_ERROR = -1;
-
-	private int[] numbers = new int[STACK_CAPACITY];
+import edu.kis.vh.nursery.collection.IntArrayStack;
+import edu.kis.vh.nursery.collection.IntLinkedList;
 
-	private int total = EMPTY_STACK_ERROR;
+public class DefaultCountingOutRhymer {
+	IntArrayStack stack = new IntArrayStack();
+	IntLinkedList list = new IntLinkedList();

```

Comment:
The goal of the lab was to make using IntLinkedList and IntArrayStack interchangeable.